### ISSUE TITLE
Implement arrayLeftRotate

### ISSUE THREAD

**Issue by s22su** [rgehan/hacktoberfest-2k18-katas#243]
# arrayLeftRotate

### Link to the PR that adds this test case:
#242

### Where to implement:
`src/arrayLeftRotate.js`

### Where are the tests:
`src/arrayLeftRotate.test.js`

### Description (optional):
Rotate array elements from left to right n times.
Example: array = [1, 2, 3, 4], n = 2;
Steps: [1, 2, 3, 4] -> [2, 3, 4, 1] -> [3, 4, 1, 2]
Result: [3, 4, 1, 2]
Return null if passed element is not array.

----

# If you want to take on this issue:
- Verify that no one is working on it (if it's labelled `assigned` for example)
- Comment here and ask to be assigned the issue
- Create a Pull Request with the following
  - A fixed implementation of the function
  - A new kata (see the README.md for more details)
  
:warning: If your PR is not following the above rules, it will be closed immediately, and tagged as `invalid`, meaning that it won't count in your Hacktoberfest's PRs :warning:

**Comment by piksu** [comment]
I think the last step of the description should be [3,4,1,2] instead of [3,4,1,3]? Same in the "Result: [3, 4, 1, 3]".

**Comment by s22su** [comment]
@piksu thank you, updated description. Will check my PR also.

**Comment by rgehan** [comment]
@piksu do you want it?

**Comment by piksu** [comment]
@rgehan sure, I just thought that others should have a chance to grab these since I've already done one kata already. However we won't be running out of the katas since new ones pops out as soon as one is finished. 👍

**Comment by KumarManoj-S** [comment]
Can it take this?

**Comment by rgehan** [comment]
It's all yours

/assign @piksu

**Comment by rgehan** [comment]
It's all yours

/assign @piksu

**Comment by rgehan** [comment]
Hey @piksu,

It looks like this issue has been inactive for at least 24 hours.

An issue is considered inactive if:
- You, the assignee, didn't comment on it during the period described above
- You, the assignee, didn't open a PR related to the issue (for it to be considered as related, the ID of the issue needs to appear either in the title of the PR, or in its body. Don't forget the # symbol before).

If you think this is an error, please ping me in your reply.

/reset

##### This was automatically generated by a bot

**Comment by MikeBudimon** [comment]
I have implemented and tested the function locally and it worked without problems but when I run the test case it fails two times. After that I looked at the test case(arrayLeftRotate.test.js) and I think it has two small errors at line 9 and 13:

- `expect(arrayLeftRotate([5, 6, 3, 4], 2), 2).toEqual([3, 4, 5, 6]);` should be `expect(arrayLeftRotate([5, 6, 3, 4], 2)).toEqual([3, 4, 5, 6]);`

- `expect(arrayLeftRotate('hello', 3)).toEqual(0);` should be `expect(arrayLeftRotate('hello', 3)).toEqual(null);`

What is the next step? Should I change the test case or @s22su should do it?
